exercises/isbn-verifier/example.js

```diff
@@ -0,0 +1,17 @@
+export default class ISBN {
+
+  constructor(isbn) {
+    this.isbn = isbn.replace(/-/g, '');
+  }
+
+  isValid() {
+    if (!this.isbn.match(/^(\d{9}[\dxX])$/)) return false;
+
+    const digits = this.isbn.split('');
```

Comment:
Perhaps use the ES6 spread operator:

```javascript
const digits = [...this.isbn];
```

Reply:
Changed in a commit. Using the spread operator is indeed the correct syntax in this instance.